src/test/java/uk/gov/service/notify/NotificationClientTest.java

```diff
@@ -52,7 +52,7 @@ public void testCreateNotificationClient_withSingleApiKeyServiceIdAndProxy() {
     @Test
     public void testCreateNotificationClientSetsUserAgent() {
         NotificationClient client = new NotificationClient(combinedApiKey, baseUrl);
-        assertEquals(client.getUserAgent(), "NOTIFY-API-JAVA-CLIENT/3.15.3-RELEASE");
+        assertEquals(client.getUserAgent(), "NOTIFY-API-JAVA-CLIENT/3.16.0-RELEASE");
```

Comment:
I've gone round and changed 3.15.3 to 3.16.0 manually using find and replace. Do I still need to run `update_version.sh`? I don't quite see what it does?

Reply:
No. Actually update_version.sh needs a change, it doesn't update the unit test. Or change the unit test to check that the user agent contains "NOTIFY-API-JAVA-CLIENT" AND "-RELEASE".

Reply:
No. Actually update_version.sh doesn't update the unit test. We should update the script or change the unit test so that it only checks that the user agent contains "NOTIFY-API-JAVA-CLIENT/*-RELEASE"

Reply:
If I change the unit test to check the regex instead does that then mean I can just completely remove the `update_version` script? I don't really understand what `mvn versions:set` is doing